app/src/views/Resources/VideoList/i18n.json

```diff
@@ -9,6 +9,8 @@
         "videoCarouselHeadingMontandon": "Montandon – the Global Crisis Data Bank",
         "videoCarouselSubHeadingMontandon": "Mapping global crises' historical data. The IFRC GO Platform uses Montandon, the Global Crisis Data Bank, the world's largest repository of information on past natural hazards and their impact.",
         "videoCarouselHeading3w": "3W (who-what-where) data collection for emergencies",
-        "videoCarouselHeading3wsub": "View this video tutorial about the 3W for emergencies tool. The video demonstrates the data collection and display activity of who-what-where data."
+        "videoCarouselSubHeading3w": "View this video tutorial about the 3W for emergencies tool. The video demonstrates the data collection and display activity of who-what-where data.",
```

Comment:
We have duplicate keys for "videoCarouselHeading3W" and "videoCarouselSubHeading3W"
Also, "videoCarouselSubHeading3WInstructional" is not used.

Reply:
@szabozoltan69 could you take a look at this and verify the correct behavior?

Reply:
3W != 3w, but I fixed it with a more verbose naming. And removed unused variable.